src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
+    private String color;
```

Comment:
Since we have a Color enum it would be better to also use it as a field type
```suggestion
    private Color color;
```